libusb/os/windows_winusb.c

```diff
@@ -1218,6 +1263,7 @@ static int init_device(struct libusb_device *dev, struct libusb_device *parent_d
 		return r;
```

Comment:
here too

maybe there're are other places, I haven't checked

Reply:
fix all where io_event is not released